lib/sidekiq/cron/schedule_loader.rb

```diff
@@ -4,11 +4,19 @@
 
 if Sidekiq.server?
   Sidekiq.configure_server do |config|
+    def load_yaml(src)
```

Comment:
can we define this method outside the block? It's seems more natural to avoid dealing with scope visibility and eventually it will be moved to something like `Utils` or `Support`.